.github/workflows/commitstyle.yml

```diff
@@ -0,0 +1,14 @@
+name: Conventional Commits
+
+on:
+  pull_request:
+    branches: [ main ]
```

Comment:
It looks like this triggers on pull requests to main... what happens if non-conventional commits are made before the PR is opened? Wouldn't this make more sense in some sort of pre-commit hook?